app/javascript/dashboard/components/ChatList.vue

```diff
@@ -1,22 +1,22 @@
 <template>
   <div
-    class="conversations-list-wrap flex-basis-clamp flex-shrink-0 overflow-hidden flex flex-col border-r rtl:border-r-0 rtl:border-l border-slate-50 dark:border-slate-800/50"
+    class="flex flex-col flex-shrink-0 overflow-hidden border-r conversations-list-wrap flex-basis-clamp rtl:border-r-0 rtl:border-l border-slate-50 dark:border-slate-800/50"
```

Comment:
@scmmishra Removing `conversations-list-wrap` class name needs some effort because it's used as a wrapper class name in `rtl.scss` file for some RTL styles.